src/js/components/products/ProductsListFilters.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect, useReducer } from 'react';
+
+import PropTypes from 'prop-types';
+
+import FilterForm from 'components/Filter/FilterForm';
+import CheckboxField from 'components/form-elements/CheckboxField';
+import FilterSelectField from 'components/form-elements/FilterSelectField';
+import { fetchProductsCatalogs, fetchProductsCategories, fetchProductsTags } from 'components/products/actions';
+import productsFiltersReducer from 'components/products/reducers/productsFiltersReducer';
+
+const filterFields = {
+  categoryId: {
+    type: FilterSelectField,
+    attributes: {
+      valueKey: 'id',
+      filterElement: true,
+      placeholder: 'Category',
+      showLabelTooltip: true,
+      multi: true,
+    },
+    getDynamicAttr: ({ categories }) => ({
+      options: categories,
+    }),
+  },
+  includeCategoryChildren: {
+    type: CheckboxField,
+    label: 'react.productsList.includeSubcategories.label',
+    defaultMessage: 'Include all products in all subcategories',
+    attributes: {
+      filterElement: true,
```

Comment:
what does the attribute `filterElement: true` do for  `CheckboxField` ?

Reply:
it is added to filter form elements not to have to play around with position styles included in `form-utils`, so it is just to position filter elements, nothing functional.

Reply:
that's right I remember now. The naming implies that it has something to do with filtering, kind of misleading